geemusic/intents/selection.py

```diff
@@ -1,9 +1,34 @@
-from flask_ask import statement, audio
+from flask_ask import statement, audio, question
 from os import environ
 from geemusic import ask, app, queue
 from geemusic.utils.music import GMusicWrapper
 from fuzzywuzzy import fuzz
 
+@ask.launch
+def login():
+    text = 'Welcome to Gee Music. Try asking me to play a song or start a playlist'
+    prompt = 'For example say, play music by A Tribe Called Quest'
+    return question(text).reprompt(prompt)
+
+@ask.intent("AMAZON.HelpIntent")
+def help():
+    text = ''' Here are some things you can say:
+                Play artist Radiohead,
+                Play songs by LCD Soundsystem,
+                Play music by A Tribe Called Quest,
+                Play the album Science For Girl,
+                Play the song Fitter Happier,
+                Start a radio station for artist Weezer,
+                Play some music (plays your I'm Feeling Lucky station),
+                Start playlist Dancy Party, 
+                
+                Of course you can also say skip, previous, shuffle, and more of alexa's music commands or, stop, if you're done.
```

Comment:
Anything you would take off? I wanted to make sure it had some of the more basic commands.